website/docs/docs/cloud/connect-data-platform/about-connections.md

```diff
@@ -20,9 +20,13 @@ dbt Cloud can connect with a variety of data platform providers including:
 - [Starburst or Trino](/docs/cloud/connect-data-platform/connect-starburst-trino)
 - [Teradata](/docs/cloud/connect-data-platform/connect-teradata) <Lifecycle status="Preview" />
 
-You can connect to your database in dbt Cloud by clicking the gear in the top right and selecting **Account Settings**. From the Account Settings page, click **+ New Project**.
+You can connect to your database in dbt Cloud by:
 
-<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choose-a-connection.png" title="Choose a connection"/>
+1. Click your name at the bottom left of your account
```

Comment:
The way you've written it here is totally valid and 100% a way that a user can navigate to the settings, but it might be beneficial to the user to lead them there via the Account settings workflow
```suggestion
1. Click your account name at the bottom of the left-side menu and click **Account settings**
```